config/kubernetes/sig-testing/teams.yaml

```diff
@@ -30,6 +31,7 @@ teams:
       sig-testing-leads:
         description: sig-testing chairs and technical leads
         members:
+        - aojea
```

Comment:
Can we add the others as well?

Reply:
ok, let me close this and send multiple PRs